bin/common.sh

```diff
@@ -88,7 +88,7 @@ function download_mysql_jdbc_jar(){
   echo "Mysql jdbc jar is downloaded and put in the path of submarine/lib."
 }
 
-JAVA_OPTS+=" ${SUBMARINE_SERVER_JAVA_OPTS} -Dfile.encoding=UTF-8 ${SUBMARINE_SERVER_MEM}"
+JAVA_OPTS+=" -Dfile.encoding=UTF-8"
```

Comment:
It's because `${SUBMARINE_SERVER_JAVA_OPTS}` and `${SUBMARINE_SERVER_MEM}"` are duplicated, so they need to be deleted, right?

Reply:
Because they are added in submarine-daemon.sh and submarine.sh. So we need to delete them to avoid deplications.